plugins/rapid7_insight_agent/unit_test/responses/get_all_agents_by_ip.resp

```diff
@@ -140,6 +140,55 @@
                 }
               }
             }
+          },
```

Comment:
should we update the unit tests to show an example of the returned location info of the agents?

Reply:
Yes, sure 😄